index.html.slim

```diff
@@ -15,7 +15,7 @@ description: The simple, modern & productive way to build apps and infrastructur
         a.spotlight-item href='http://developers.redhat.com/promotions/devnation-discount-code'
           img(src="http://static.jboss.org/images/rhd/promo/RHDev_homepromo_devnation_30percent_18apr2016.png" alt="Get 30% off registration for DevNation")
           span Join RHD and get a special 30% discount code.
-        a.spotlight-item href='http://developers.redhat.com/register'
+        a.spotlight-item href='http://developers.redhat.com/promotions/devnation-pass-giveaway/'
```

Comment:
All the links should be in the form of `#{site.base_url}/path/to/resource` so they go to the correct location based on the environment.